src/components/plots/areaPlot.ts

```diff
@@ -32,8 +30,25 @@ export module Plots {
       this._lineDrawers.values().forEach((d) => d.setup(this._renderArea.append("g")));
     }
 
+    /**
+     * Gets the AccessorScaleBinding for Y0.
+     */
     public y0(): Plots.AccessorScaleBinding<number, number>;
+    /**
+     * Sets Y0 to a constant number, or to the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} y0
+     * @returns {Area} The calling Area Plot.
+     */
     public y0(y0: number | Accessor<number>): Area<X>;
+    /**
+     * Sets Y0 to a scaled constant value or scaled result of an Accessor.
+     * The provided Scale will account for the values when autoDomain()-ing.
```

Comment:
"The provided Scale will account for the values when autoDomain()-ing." sounds really vague.  Probably be more specific that the `Plot` will react if the y0Scale undergoes changes.